metadata.json

```diff
@@ -1,39 +1,70 @@
 {
   "name": "MiamiOH-shibboleth_idp",
-  "version": "1.3.0",
+  "version": "1.4.0",
   "author": "Dirk Tepe",
   "summary": "Manage Shibboleth IdP",
-  "license": "GPL-3.0+",
```

Comment:
DId you or pdk remove this +

Reply:
seems like GPL-3.0+ is depreciated

https://spdx.org/licenses/

Reply:
This is the new equiv:
GPL-3.0-or-later

Reply:
I removed due to deprecation,  will update to suggested